README.md

```diff
@@ -185,12 +194,10 @@ Anyone who has been active in the foundation or one of its member projects, as d
 - Rich Trott ([@Trott](https://github.com/Trott))
 - Rifa Achrinza ([@achrinza](https://github.com/achrinza))
 - Ruy Adorno ([@ruyadorno](https://github.com/ruyadorno))
-- Sara Chipps ([@sarajo](https://github.com/sarajo))
```

Comment:
Also Sara should stay in that list

Reply:
whoops, I thought they were a voting member.  good catch.  (although it sounds like this is superseded by Jordan's comment anyway)